cypress/README.md

```diff
@@ -0,0 +1,214 @@
+2. Pull requests will be tested against environment before merging to main codebase
+3. Pull request's owner must add **RFR** label once pull request is ready to test against environment
+4. After adding **RFR** label, the PR owner should force push the pull request to trigger GitHub action
+5. In some cases, the reviewer wants to trigger the PR test GH action but they can't force push to the PR branch. Hence added feature of triggering GH action manually. Steps to trigger GH action for PR testing manually:
+   - Go to Actions tab
+   - Click on GH action(Pull request test)
+   - Click on run workflow dropdown
+   - select PR branch
+   - Click on button Run workflow
+
+## Tag based test execution
+
+This repository uses the package [cypress-tags](https://www.npmjs.com/package/cypress-tags) to slice up test runs.
+
+1. To run tests based on tags, use below command
+
+   ```sh
+   CYPRESS_INCLUDE_TAGS=@tagName npx cypress run
+   ```
+
+2. To run multiple tags(tiers) in a single run, provide tag names separated by commas
+   ```sh
+   CYPRESS_INCLUDE_TAGS=@tier1,@tier2 npx cypress run
+   ```
+
+## Running tests locally with Konveyor operator installed to minikube
+
+1. Install minikube on your local machine by referring to [docs](https://minikube.sigs.k8s.io/docs/start/)
+
+1. Setup recommended minikube configurations and start minikube with the ingress addon
+
+   ```sh
+   minikube config set memory 10240
+   minikube config set cpus 4
+   minikube start --addons=ingress
+   ```
+
+1. Install kubectl if needed, or substitute `minikube kubectl --` for all `kubectl` commands.
+
+1. Install [OLM manually](https://github.com/operator-framework/operator-lifecycle-manager/releases)
+
+   ```sh
+   curl -L https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.28.0/install.sh -o install.sh
+   chmod +x install.sh
+   ./install.sh v0.28.0
+   ```
+
+1. Install the Konveyor operator ([operator install](https://github.com/konveyor/operator?tab=readme-ov-file#konveyor-operator-installation-on-k8s)) to the default namespace **konveyor-tackle**
+
+   ```sh
+   kubectl apply -f https://raw.githubusercontent.com/konveyor/operator/main/tackle-k8s.yaml
+   ```
+
+1. Create a Tackle CR (setting the spec values as needed)
+
+   ```sh
+   cat << EOF | kubectl apply -f -
+   kind: Tackle
+   apiVersion: tackle.konveyor.io/v1alpha1
+   metadata:
+     name: tackle
+     namespace: konveyor-tackle
+   spec:
+     feature_auth_required: false
+   EOF
+   ```
+
+1. Wait for the tackle ingress to become available
+
+   ```sh
+   kubectl wait -n konveyor-tackle ingress/tackle --for=jsonpath='{.status.loadBalancer.ingress[0]}' --timeout=600s
+   ```
+
+1. Once the Konveyor UI ingress is running, tests can be run against this local instance, with video captures of the tests turned on, using the command
+   ```sh
+   CYPRESS_baseUrl=https://$(minikube ip) npx cypress run --config video=true
+   ```
```

Comment:
This can be reworked by referencing https://github.com/konveyor/tackle2-ui/blob/main/docs/local-minikube-setup.md